proxy/envoy/config.go

```diff
@@ -267,6 +268,21 @@ func buildOutboundHTTPRoutes(instances []*model.ServiceInstance, services []*mod
 					routes = append(routes, buildDefaultRoute(cluster))
 				}
 
+				if service.External {
+					for _ , route := range routes {
+						for _ , cluster := range route.clusters {
+							cluster.ServiceName = ""
```

Comment:
We cannot use SDS for egress proxies. Because SDS cannot serve DNS names. Only IP/ports are allowed. If we convince Matt that SDS could allow logical_dns and strict_dns, then we might use SDS for egress as well.